src/hosted_db/sqlite.rs

```diff
@@ -36,3 +47,35 @@ pub fn run_sqlite_query(path: &PathBuf, query: &str) -> Result<QueryResult, AybE
         rows: results,
     })
 }
+
+pub async fn run_sqlite_query(
+    path: &PathBuf,
+    query: &str,
+    isolation: &Option<AybConfigIsolation>,
+) -> Result<QueryResult, AybError> {
+    match isolation {
```

Comment:
Most of the checks here can be replaced with early returns, which would make the code more readable. :)

Reply:
Thank you for this feedback. Aside from the `match`, which we'll discuss below, are there others to consider?

Reply:
Nope :)